<?xml version="1.0"?>
<launch>
    <param name="serverNameOrIP" value="192.168.1.3" />
    <group ns="/kinect2">
        <param name="rgb_frame" value="/head/kinect2/rgb"/>
        <param name="depth_frame" value="/head/kinect2/depth"/>
        <param name="ir_frame" value="/head/kinect2/depth"/>
        
        <node name="startRGB" pkg="k2_client" type="startRGB"/>
        <node name="startDepth" pkg="k2_client" type="startDepth" />
        <node name="startIR" pkg="k2_client" type="startIR"/>
        <node name="startBody" pkg="k2_client" type="startBody"/>
        <node name="startAudio" pkg="k2_client" type="startAudio"/>
    </group>
</launch>
